### PR TITLE
CB-13323: Cordova-Android 6.3.0 Release Announcement

### PR DESCRIPTION
**Pull request by infil00p**
### Platforms affected
Docs

### What does this PR do?
This has the Cordova-Android 6.3.0 Release Post

### Checklist
- [ ] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [ ] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [ ] Added automated test coverage as appropriate for this change.

### CODE REVIEW

File: www/_posts/2017-09-27-android-release.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+This release now targets the latest Android API level of API 26 and has fixed issues related to the **Android SDK Tools 26.0.2** release. **Google** changed how the Android emulator was executed, causing errors when deploying to the emulator. 
+
+This release contains the integration of `cordova-plugin-compat`, so please remove that plugin from projects once you update to this version of `cordova-android`.
+
+To upgrade:
+
+    npm install -g cordova
+    cd my_project
+    cordova platform update android@6.2.1
```
Comment (janpio) [inline www/_posts/2017-09-27-android-release.md]: typo, should be @6.3.0
(same problem below)

File: www/_posts/2017-09-27-android-release.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+* Revert [CB-12015](https://issues.apache.org/jira/browse/CB-12015) initial-scale values less than 1.0 are ignored on **Android**
+* [CB-12730](https://issues.apache.org/jira/browse/CB-12730) The Cordova Compatibility Plugin is now integrated into `cordova-android`
+* [CB-12453](https://issues.apache.org/jira/browse/CB-12453) Remove unnecessary double quotes from .bat files which are the causes of crash if project path contains spaces
+* [CB-13031](https://issues.apache.org/jira/browse/CB-13031) Fix bug with case-sensitivity of **Android**-packageName
+* [CB-10916](https://issues.apache.org/jira/browse/CB-10916) Support display name for **Android**
+* [CB-12423](https://issues.apache.org/jira/browse/CB-12423) make explicit JDK 1.8 or greater is needed in the `README`, we require 1.8 for compilation, but do not have 1.8 Java features yet
+* [CB-12605](https://issues.apache.org/jira/browse/CB-12605) In **Windows** get **Android** studio path from the registry
+* [CB-12617](https://issues.apache.org/jira/browse/CB-12617) : removed node0.x support for platforms and added engineStrict
+
+#
```
Comment (janpio) [inline www/_posts/2017-09-27-android-release.md]: Should this be here?

Comment (stevengill) [reply www/_posts/2017-09-27-android-release.md]: yeah, looks like an extra `#`

**Comment by audreyso** [comment]
Merged! Please close :) Thank you!

**Pull request closed without merging**